src/Microsoft.ML.Ensemble/Trainer/Binary/EnsembleTrainer.cs

```diff
@@ -70,6 +73,12 @@ public EnsembleTrainer(IHostEnvironment env, Arguments args)
             Combiner = args.OutputCombiner.CreateComponent(Host);
         }
 
+        public EnsembleTrainer(IHostEnvironment env, Arguments args, PredictionKind predictionKind)
```

Comment:
>  public EnsembleTrainer(IHostEnvironment env, Arguments args, PredictionKind predictionKind) [](start = 6, length = 93)

in the spirit of reducing public ctor, would it make sense to add the PredictionKind as an optional param, or that would be confusing?

Reply:
I made it private :-).

---
In reply to: [227114506](https://github.com/dotnet/machinelearning/pull/1305#discussion_r227114506) [](ancestors = 227114506)